cldf/cldf-metadata.json

```diff
@@ -11,37 +11,37 @@
     "dc:related": null,
     "dc:source": "sources.bib",
     "dc:title": "Wordlists in Selected Languages of Nepal",
-    "dcat:accessURL": "https://github.com/lexibank/halenepal",
+    "dcat:accessURL": "git@github.com:lexibank/halenepal",
```

Comment:
@chrzyki I admit that using the git remote URL as access URL is somewhat misuse - but since we do, could you switch to cloning via https?

Reply:
Done!